index.html

```diff
@@ -21,285 +21,338 @@
-                        <div class="project">
-                            <h2>Data Dashboard Healthcare</h2>
-                            <p>A daily selection of privately personalized reads
-                                no accounts or sign-ups required. has been the industry's standard
-                            </p>
-                        
-                             <a href="#" class="project-btn">See project</a>
-                        
+
+            </section>
+
+
+
+
+
+            <section class="work-container" id="portfolio">
+                <h2 class="title"> My Recent Works</h2>
+                <div class="feature-project">
+
+                    <div class="phimg">
+                        <img src="./images/imgwork.png" alt="placeholder-img">
+                    </div>
+
+                    <div class="text-work">
+                        <h2 class="project-title">Multi-Post Stories</h2>
+                        <p class="description">A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. has been the
+                            industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                            standard dummy text.
+                        </p>
+
+                        <div class="languages">
+                            <ul>
+                                <li>css</li>
+                                <li> html</li>
+                                <li class="bootstrap">bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+
+                        <a href="">See Project</a>
+                    </div>
+                </div>
+
+
+                <div class="projects">
+                    <div class="project">
+                        <h2>Data Dashboard Healthcare</h2>
+                        <p>A daily selection of privately personalized reads
+                            no accounts or sign-ups required. has been the industry's standard
+                        </p>
+
+                        <button class="project-btn" onclick="modal(0)">See project</button>
+
+                    </div>
+
+                    <div class="project">
+                        <h2>Data Dashboard Healthcare</h2>
+                        <p>A daily selection of privately personalized reads
+                            no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+
+                            <ul>
+                                <li class="project-language"> html
+                                <li>
+                                <li class="project-language">bootstrap
+                                <li>
+                                <li class="project-language"> Ruby
+                                <li>
+                            </ul>
```

Comment:
- [ ] You have done a great job adding an array containing all projects in JavaScrip, however, you should not have the projects section inside your HTML file directly, since it should be generated by javascript from the projects array as mentioned in the project requirements 👇 

![image](https://user-images.githubusercontent.com/55465133/131916383-5d47d9cf-129f-4a0e-9b88-8f0d0f42a9cd.png)

With this adding projects in the future will be easy. kindly let's fix this issue, I am sure you get make it 😉